ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -317,19 +327,21 @@ def timeout(self, func, seconds, message):
 
     def generate_vlan_servers(self):
         vlan_host_map = defaultdict(dict)
-        vlan_ip_range = self.test_params['vlan_ip_range']
-
-        _, mask = vlan_ip_range.split('/')
-        n_hosts = min(2**(32 - int(mask)) - 3, self.max_nr_vl_pkts)
+        self.nr_vl_pkts = 0
```

Comment:
>nr_vl_pkts [](http://example.com/codeflow?start=13&length=10)

The var name is hard to read. Could you add some code comment?

Reply:
Added comments to clarify this is the number of packets from upper layer.